api/dao/containerutil.py

```diff
@@ -0,0 +1,28 @@
+import bson.objectid
```

Comment:
This one broke the camel's back of needing a better place to put functions that act on specific containers and interact with the db. Suggesting this module as a midstep between ad hoc as needed methods outside of the `doa` package and having container-specific modules.